<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Execut_window</class>
 <widget class="QDialog" name="Execut_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>601</width>
           <height>441</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>601</width>
           <height>441</height>
          </size>
         </property>
         <widget class="QPushButton" name="pBt_ignore">
          <property name="geometry">
           <rect>
            <x>430</x>
            <y>350</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>跳过</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_Position">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>150</y>
            <width>281</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_Name">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>90</y>
            <width>281</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_changeNewAgentia">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>250</y>
            <width>111</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>替换为：</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>160</y>
            <width>51</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>位置：</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pBt_cancal">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>350</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>取消</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_changeVolume">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>290</y>
            <width>281</width>
            <height>31</height>
           </rect>
          </property>
          <property name="placeholderText">
           <string>如果更新数据，必须输入；不输入视为信息正确</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>220</y>
            <width>51</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>容量：</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_RoleStatus">
          <property name="geometry">
           <rect>
            <x>350</x>
            <y>50</y>
            <width>191</width>
            <height>21</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_oddDose">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>290</y>
            <width>71</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>剩余容量：</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pBt_next">
          <property name="geometry">
           <rect>
            <x>330</x>
            <y>350</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>操作完成</string>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>43</x>
            <y>100</y>
            <width>71</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>试剂名：</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_title">
          <property name="geometry">
           <rect>
            <x>193</x>
            <y>50</y>
            <width>71</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_Volume">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>210</y>
            <width>281</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QComboBox" name="CMB_V">
          <property name="geometry">
           <rect>
            <x>410</x>
            <y>290</y>
            <width>61</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Adobe Arabic</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>ml</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>g</string>
           </property>
          </item>
         </widget>
         <widget class="Line" name="line">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>600</width>
            <height>3</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>600</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>600</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="Line" name="line_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>3</width>
            <height>440</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>440</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>400</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
         <widget class="Line" name="line_4">
          <property name="geometry">
           <rect>
            <x>600</x>
            <y>0</y>
            <width>3</width>
            <height>440</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>440</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>400</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
         <widget class="Line" name="line_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>440</y>
            <width>600</width>
            <height>3</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>600</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>600</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
